src/network/networkd-gperf.gperf

```diff
@@ -23,3 +24,5 @@ Network.SpeedMeterIntervalSec, config_parse_sec,                       0,
 Network.ManageForeignRoutes,   config_parse_bool,                      0,          offsetof(Manager, manage_foreign_routes)
 DHCP.DUIDType,                 config_parse_duid_type,                 0,          offsetof(Manager, duid)
 DHCP.DUIDRawData,              config_parse_duid_rawdata,              0,          offsetof(Manager, duid)
+MPTCP.LimitSubflows,           config_parse_uint32,                    0,          offsetof(Manager, mp_tcp_subflows)
+MPTCP.LimitAddress,            config_parse_uint32,                    0,          offsetof(Manager, mp_tcp_add_addr_accepted)
```

Comment:
Sorry, in the previous review, I said `config_parse_uint32()` should be used for them, but the values must be less than or equal to 8 (`MPTCP_PM_ADDR_MAX`). So, please re-introduce e.g. `config_parse_mptcp_limit()` and check whether the value is in the range.

Reply:
It's also required for the logging.